src/crs_linter/rules/ordered_actions.py

```diff
@@ -0,0 +1,100 @@
+from crs_linter.lint_problem import LintProblem
+from crs_linter.utils import get_id
+from crs_linter.rule import Rule
+
+ACTIONS_ORDER = [
+    "id",  # 0
+    "phase",  # 1
+    "allow",
+    "block",
+    "deny",
+    "drop",
+    "pass",
+    "proxy",
+    "redirect",
+    "status",
+    "capture",  # 10
+    "t",
+    "log",
+    "nolog",
+    "auditlog",
+    "noauditlog",
+    "msg",
+    "logdata",
+    "tag",
+    "sanitisearg",
+    "sanitiserequestheader",  # 20
+    "sanitisematched",
+    "sanitisematchedbytes",
+    "ctl",
+    "ver",
+    "severity",
+    "multimatch",
+    "initcol",
+    "setenv",
+    "setvar",
+    "expirevar",  # 30
+    "chain",
+    "skip",
+    "skipafter",
+]
+
+
+class OrderedActions(Rule):
+    """Check that actions are in the correct order."""
+
+    def __init__(self):
+        super().__init__()
+        self.success_message = "Action order check ok."
+        self.error_message = "Action order check found error(s)"
+        self.error_title = "Action order check"
+        self.args = ("data",)
+
+    def check(self, data):
+        chained = False
+
+        for d in data:
+            if "actions" in d:
+                max_order = 0  # maximum position of read actions
+                if not chained:
+                    current_rule_id = get_id(d["actions"])
```

Comment:
Here you don't use variable `current_rule_id`, you should remove it or add it to message.

Reply:
Added